packages/story-editor/src/components/colorPicker/basicColorPicker.js

```diff
@@ -28,32 +28,48 @@ import {
   BUTTON_SIZES,
   BUTTON_TYPES,
   BUTTON_VARIANTS,
+  localStore,
+  LOCAL_STORAGE_PREFIX,
 } from '@web-stories-wp/design-system';
 import { __ } from '@web-stories-wp/i18n';
+import { useState } from '@web-stories-wp/react';
 
 /**
  * Internal dependencies
  */
 import useStory from '../../app/story/useStory';
+import useEyedropper from '../eyedropper';
 import { BASIC_COLORS } from './constants';
 import Header from './header';
 import BasicColorList from './basicColorList';
+import ConfirmationDialog from './confirmationDialog';
+import useDeleteColor from './useDeleteColor';
 
 const Body = styled.div`
   display: flex;
   flex-direction: column;
   margin: 8px 16px 16px;
-  gap: 16px;
+`;
+
+const SavedColors = styled.div`
+  border-bottom: 1px solid ${({ theme }) => theme.colors.border.defaultNormal};
+  padding-bottom: 24px;
```

Comment:
FYI when reviewing: There were some pixel inconsistencies in the Color Picker in Figma (e.g. partial pixels, having 23px in one place for padding and 24px in other place, etc.) -- I chose what made the most sense for consistency.